test/net/sourceforge/kolmafia/KoLAdventureValidationTest.java

```diff
@@ -1383,6 +1397,36 @@ void cannotAdventureInUnknownNewOrchardZone() {
         assertThat(UNKNOWN.canAdventure(), is(false));
       }
     }
+
+    @CartesianTest
+    void preparingForChamberUsesGland(
+        @CartesianTest.Enum Chambers chamber,
+        @Values(booleans = {true, false}) final boolean hasGland,
```

Comment:
Consistency: previous tests are "haveEffect" and "haveGland".
This one is "hasEffect" and "hasGland".

Which is it: "I have the effect" or "the character has the effect"?

Reply:
I will rename this one :)